Development/IDS_Aedes_example.ids

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<ids:ids xmlns:ids="http://standards.buildingsmart.org/IDS" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://standards.buildingsmart.org/IDS http://standards.buildingsmart.org/IDS/0.9.6/ids.xsd">
+<ids:ids xmlns:ids="http://standards.buildingsmart.org/IDS" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://standards.buildingsmart.org/IDS http://standards.buildingsmart.org/IDS/0.9.7/ids.xsd">
     <ids:info>
         <ids:title>AedesUVIP-ILS Window use-case example</ids:title>
         <ids:version>1.0</ids:version>
     </ids:info>
     <ids:specifications>
-        <ids:specification ifcVersion="IFC4" name="Beglazing" description="Glazing in a window" minOccurs="0" maxOccurs="unbounded">
-            <ids:applicability>
+        <ids:specification ifcVersion="IFC4" name="Beglazing" description="Glazing in a window" >
+            <ids:applicability minOccurs="0" maxOccurs="unbounded">
```

Comment:
I have never understood what `minOccurs` and `maxOccurs` mean in an `applicability `statement.

Reply:
There was a long debate about the meaning of min/maxOccurs settings in `specification`. The decision from an engineering point of view was to ask for a specific number of instances that can be selected by `applicability `where ALL instances must fulfill specified requirements. This is the reason to move from `specification `to `applicability`, because this cardinality restriction is purely for number of instances that fulfill the `applicability `statement.

In contrast to that, we did not see a use case asking for a specific number of instances that are selected by applicability AND fulfill specified requirements.
 
Anyhow, most generic EIRs will use min=0 and max=unbounded, meaning that ALL instances fulfilling the applicability settings should fulfill our requirements. There are only few examples where other settings make sense and most of them seem to be addressed either in the IFC schema (1 instance of IfcProject) or implementer agreements (1 instance of IfcSite/IfcBuilding). Other examples like "I need 10 office rooms in my building" are more project specific.